src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -551,6 +551,9 @@ private void OnNodeViewMouseLeave(object sender, MouseEventArgs e)
         {
             ViewModel.ZIndex = oldZIndex;
 
+            //Watch nodes doesn't have Preview so we should avoid to use any method/property in PreviewControl class due that Preview is created automatically
+            if (ViewModel.NodeModel != null && ViewModel.NodeModel is CoreNodeModels.Watch) return;
+
```

Comment:
another Idea would be to stop loading controls that are not immediately visible to the user (like context menu stuff, tooltips etc) These could be loaded on demand when the user interacts with the UI.

Reply:
@pinzart90 I think for this case we will need a specific Jira task since I noticed that just the PreviewControl is being used in several places and due to the current implementation of this property, when you use it automatically creates the instance.
For example in this piece of code when using PreviewControl.StaysOpen, the instance is created automatically:
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/cbc75303-6f3d-4870-9d54-a54fe225422d)


This is the declaration of the Property and the references in code
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/af034f2a-05b3-4221-be6f-514a0170d481)

Reply:
Yep, a lot of code assumes the existence of the PreviewControl.  We should create a task for it if we can say for sure it would benefit performance.
It should either be properly refactored to be optional or create it hidden for certain node types (so that it does not contribute to the layout tree calculations) Not sure about the perf results of either.